row_group_test.go

```diff
@@ -106,210 +106,216 @@ func testSeekToRow(t *testing.T, newRowGroup func([]Person) parquet.RowGroup) {
 	}
 }
 
-var mergeRowGroupsTests = [...]struct {
+type mergeRowGroupTestCase struct {
 	scenario string
 	options  []parquet.RowGroupOption
 	input    []parquet.RowGroup
 	output   parquet.RowGroup
-}{
-	{
-		scenario: "no row groups",
-		options: []parquet.RowGroupOption{
-			parquet.SchemaOf(Person{}),
-		},
-		output: sortedRowGroup(
-			[]parquet.RowGroupOption{
+}
+
+// Use a function instead of an array because otherwise the sortedRowGroup()
+// calls will exercise the code even if you're not running these test cases.
+var mergeRowGroupsTests = func() []mergeRowGroupTestCase {
```

Comment:
Should this just be a function instead of a global variable?

Reply:
yeah good call